CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 
 * Only print browser load errors once per browser.
 
+* Gracefully timeout when attempting to deserialize a test suite.
```

Comment:
Nit: "time out". It's a noun without a space, and a verb with one.

![giphy 1](https://cloud.githubusercontent.com/assets/188/26702216/e718d1f8-46d8-11e7-93c0-0185ed05a623.gif)